packages/next/lib/has-necessary-dependencies.ts

```diff
@@ -70,18 +70,15 @@ export async function hasNecessaryDependencies(
   const removalLintMsg =
     `\n\n` +
     (lintDuringBuild
-      ? `If you do not want to run ESLint during builds, run ${chalk.bold.cyan(
-          'next build --no-lint'
-        )}` +
+      ? `If you do not want to run ESLint during builds, ` +
         (!!eslintrcFile
-          ? ` or remove the ${chalk.bold(
+          ? `remove the ${chalk.bold(
               basename(eslintrcFile)
-            )} file from your package root.`
+            )} file from your package root`
           : pkgJsonEslintConfig
-          ? ` or remove the ${chalk.bold(
-              'eslintConfig'
-            )} field from package.json.`
-          : '')
+          ? `remove the ${chalk.bold('eslintConfig')} field from package.json`
+          : '') +
+        ' or disable it in next.config.js.\n\nSee https://nextjs.org/docs/api-reference/next.config.js/ignoring-eslint.'
```

Comment:
Should we only recommend this config option instead of recommending deleting the configuration? Would make it easier to toggle removing a bad config from editing right away.

Reply:
Yeah that probably makes sense (I only recently added that config option)

Fixed here: https://github.com/vercel/next.js/pull/25917